src/components/TodoContainer.jsx

```diff
@@ -86,10 +81,13 @@ function TodoContainer({ tableName, baseId, apiKey }) {
             const data = await response.json();
 
             // Transforming Airtable records into the todoList format
-            const todos = data.records.map(todo => ({
-                id: todo.id,
-                title: todo.fields.title,
-                createdTime: todo.createdTime,
+            const todos = data.records
+                .filter(todo => todo.fields.list === currentTodoList)
```

Comment:
when I removed `=== currentTodoList` everything works on my side, probably I need your environment setup.

Reply:
I fixed functionality in next commit. Still, the table needs to have the additional column "list" with the name of TodoList for associated todos.